javaparser-core/src/main/java/com/github/javaparser/PositionUtils.java

```diff
@@ -132,8 +132,7 @@ public static boolean nodeContains(Node container, Node contained, boolean ignor
             if (bl>contained.getBeginLine()) return false;
             if (bl==contained.getBeginLine() && bc>contained.getBeginColumn()) return false;
             if (container.getEndLine()<contained.getEndLine()) return false;
-            if (container.getEndLine()==contained.getEndLine() && container.getEndColumn()<contained.getEndColumn()) return false;
-            return true;
+            return !(container.getEndLine() == contained.getEndLine() && container.getEndColumn() < contained.getEndColumn());
```

Comment:
Tomorrow I'll write a bunch of tests for Range and Position, I'll pick this up too.